packages/site-demo/content/product/components/checkbox/index.mdx

```diff
@@ -4,6 +4,14 @@
 
 <DemoBlock demo="default" withThemeSwitcher />
 
+### Accessibility concerns
+
+If you use the `Checkbox` component with a label, there is nothing more to add to make this component accessible.
+
+If you have no label, then, **you should provide a label**, you have 2 options to do so:
```

Comment:
Maybe we can reword a bit to avoid repetition ?
```suggestion
If you have no visible label, then **you should provide alternatives**, you have 2 options to do so:
```